kumascript/src/api/web.js

```diff
@@ -59,14 +83,22 @@ module.exports = {
             // So those with titles we ignore.
             suggested = suggested.replace(/\//g, ".");
           }
-          flaw = this.env.recordNonFatalError(
-            "redirected-link",
-            `${hrefpath} redirects to ${page.url}`,
-            {
-              current: subpath,
-              suggested,
-            }
-          );
+          if (ignoreFlawMacro) {
+            warnBrokenFlawByMacro(
+              ignoreFlawMacro,
+              href,
+              `redirects to ${page.url}`
+            );
+          } else {
+            flaw = this.env.recordNonFatalError(
+              "redirected-link",
+              `${hrefpath} redirects to ${page.url}`,
+              {
+                current: subpath,
+                suggested,
+              }
+            );
```

Comment:
Seems like we need to add the following?
```suggestion
            );
            flawAttribute = ` data-flaw-src="${util.htmlEscape(flaw.macroSource)}"`;
```